charts/osm/values.yaml

```diff
@@ -20,6 +20,11 @@ OpenServiceMesh:
     protocol: http
     token:
     role: openservicemesh
+  certmanager:
+    issuerName: osm-ca
+    issuerKind: Issuer
+    issuerGroup: cert-manager
+    caSecret: osm-ca
```

Comment:
We have been using the `osm-cabundle` secret to store the CA cert and private key. Is it possible to reuse this with CertManager?
Look in the repo for `caBundleSecretName`

Reply:
Sure thing. I wasn't sure whether to re-use this or not since the semantics are a bit different in the cert-manager case (we want just the `ca.crt` in the case for cert-manager, and tresor want's the full pk+cert bundle).

Reply:
Ah I see, would be possible to update the code such that the `private_key` can be made optional in the secret depending on the certificate manager being used?
The reason I suggest to reuse the `osm-cabundle` secret is because we reference this secret to use with other integrations, for ex. ingress. Refer to `docs/patters/ingress.md` for more details.

Reply:
Yep, that makes sense to me